pubspec.yaml

```
@@ -66,4 +66,4 @@ dev_dependencies:
 flutter:
   uses-material-design: true
   assets:
-    - assets/
\ No newline at end of file
+    - .env
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)